test/react-plugin-test.js

```diff
@@ -39,18 +39,26 @@ test('Should get DOM node by composite selector', async t => {
         .expect(await listItem2.id).eql('l1-item2');
 });
 
-test('Should not get DOM node for stateless component', async t => {
-    var stateless = await ReactSelector.with({ timeout: 100 })('StatelessComponent');
-
-    await t.expect(stateless).notOk();
+test('Should get DOM node for stateless component', async t => {
+    var stateless1 = await ReactSelector('Stateless1');
+    var stateless2 = await ReactSelector('Stateless2');
+    var stateless3 = await ReactSelector('Stateless3');
+
+    await t.expect(stateless1.textContent).ok('test');
+    await t.expect(stateless2).ok();
```

Comment:
What would you like to check here? 
If you want to get a snapshot you should call it as an async function:
```js
var stateless2 = await ReactSelector('Stateless2');  -> var stateless2 = await ReactSelector('Stateless2')();
```

Or I guess it's better to check that the element exists:
```js
await t.expect(stateless2.exists).ok();
```